From: Aurelien Jarno <aurel32@debian.org>
Date: Wed, 23 Dec 2009 08:49:00 +0100
Subject: FTBFS on GNU/kFreeBSD (bug #336898)

@DPATCH@
---
 guitune.cc |    5 +++++
 main.cc    |    6 ++++++
 2 files changed, 11 insertions(+), 0 deletions(-)

diff --git a/guitune.cc b/guitune.cc
index ba41cad..fd188d8 100644
--- a/guitune.cc
+++ b/guitune.cc
@@ -24,7 +24,12 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 #include <sys/ioctl.h>
+#if defined(__linux__)
 #include <linux/soundcard.h>
+#endif
+#if defined(__FreeBSD__) || defined(__FreeBSD_kernel__)
+#include <sys/soundcard.h>
+#endif

 #include <gtkmm/main.h>
 #include <gtkmm/table.h>
diff --git a/main.cc b/main.cc
index 001a17a..a46ce01 100644
--- a/main.cc
+++ b/main.cc
@@ -19,12 +19,18 @@
 //    Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 //
 //
+#include <cstdlib>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
 #include <unistd.h>
 #include <sys/ioctl.h>
+#if defined(__linux__)
 #include <linux/soundcard.h>
+#endif
+#if defined(__FreeBSD__) || defined(__FreeBSD_kernel__)
+#include <sys/soundcard.h>
+#endif

 #include <gtk/gtk.h>
 #include <gtkmm/window.h>
--
